contrib/kube-prometheus/jsonnet/kube-prometheus/prometheus/prometheus.libsonnet

```diff
@@ -18,6 +18,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
 
     prometheus+:: {
       name: 'k8s',
+      enableAdminAPI: 'false',
```

Comment:
jsonnet has merge semantics, therefore we don't need to specify this here at all